frontend/src/metabase/browse/databases/BrowseDatabases.tsx

```diff
@@ -52,27 +66,75 @@ export const BrowseDatabases = () => {
       <BrowseDataHeader />
       <BrowseMain>
         <BrowseSection>
-          <DatabaseGrid data-testid="database-browser">
-            {databases.map(database => (
-              <div key={database.id}>
-                <DatabaseCardLink to={Urls.browseDatabase(database)}>
-                  <DatabaseCard>
-                    <Icon
-                      name="database"
-                      color={color("accent2")}
-                      className={CS.mb3}
-                      size={32}
-                    />
-                    <Title order={2} size="1rem" lh="1rem" color="inherit">
+          <BrowseGrid data-testid="database-browser">
+            {databases &&
+              databases.length > 0 &&
+              databases.map(database => (
+                <Link to={Urls.browseDatabase(database)} key={database.id}>
+                  <DatabaseCard
+                    bg="bg-white"
+                    className={cx(CS.rounded, CS.bordered, DB.dbCard)}
+                  >
+                    <Icon name="database" color={color("accent2")} size={32} />
+                    <Title order={2} size="md" lh={1.2} color="inherit">
                       {database.name}
                     </Title>
                   </DatabaseCard>
-                </DatabaseCardLink>
-              </div>
-            ))}
-          </DatabaseGrid>
+                </Link>
+              ))}
+
+            {isAdmin && (
+              <Link to={newDatabase()} onClick={() => trackAddDatabaseDBList()}>
+                <DatabaseCard
+                  className={cx(CS.rounded, CS.bordered, DB.addCard)}
+                >
+                  <Group spacing="xs">
+                    <CardImageWrapper database={"postgres"} />
+                    <CardImageWrapper database={"mysql"} />
+                    <CardImageWrapper database={"snowflake"} />
+                  </Group>
+                  <div>
+                    <Title order={2} size="md" lh={1.2} color="inherit">
+                      {t`Add a database`}
+                    </Title>
+                    <Text
+                      color="inherit"
+                      fz="sm"
+                      lh={1}
+                      span
+                    >{t`20+ data connectors. Start exploring in minutes.`}</Text>
+                  </div>
+                </DatabaseCard>
+              </Link>
+            )}
```

Comment:
nit: but I would make this its own component like `<NewDatabaseCard />`

Reply:
Would it be ok if it lived in this same directory?
This is very often my source of confusion. At the same time we're not very consistent or not following some _strict_ component structure, but are somehow opinionated as to where should what live.

IMO, each folder (no matter how deep in the structure) should be allowed to have its own `components` if needed.
But I'd probably just do it like this for now:
```
frontend/src/metabase/browse/databases
├── BrowseDatabases.module.css
├── BrowseDatabases.tsx
├── BrowseDatabases.unit.spec.tsx
├── NewDatabaseCard
│   ├── NewDatabaseCard.tsx
│   └── index.ts
├── analytics.ts
└── index.tsx
```

WDYT?